docs/portal/registry-users/adding-users.md

```diff
@@ -0,0 +1,19 @@
+# Adding Users
```

Comment:
To avoid possible confusion, we'd better always include "registry" when referring to a registry user. Otherwise, it might be confused with "portal user".

Reply:
Done.